tests/FontLib/FontTest.php

```diff
@@ -7,12 +7,19 @@
 
 class FontTest extends TestCase
 {
-    /**
-     * @expectedException \Fontlib\Exception\FontNotFoundException
-     */
     public function testLoadFileNotFound()
     {
-        Font::load('non-existing/font.ttf');
+        // @todo when PHP 5.4 support is dropped, uncomment line below and drop
+        //       the try...catch block.
+        // $this->expectException('\Fontlib\Exception\FontNotFoundException');
+        try {
+            Font::load('non-existing/font.ttf');
+            $this->fail('Load should have failed.');
+        }
+        catch (\Fontlib\Exception\FontNotFoundException $e) {
+            // Avoid throwing a risky test error.
+            $this->assertTrue(true);
```

Comment:
You can use `expectNotToPerformAssertions()` instead.

Reply:
Not sure in PHPUnit 4? Tests need to run on PHP 5.4 too.

Reply:
Although... not sure if it's available in old phpunit versions...